vita3k/codec/src/mp3.cpp

```diff
@@ -123,6 +123,13 @@ uint32_t Mp3DecoderState::get_es_size() {
     return es_size_used;
 }
 
+void Mp3DecoderState::clear_context() {
+    int err = avcodec_close(context);
+    assert(err == 0);
+    err = avcodec_open2(context, codec, nullptr);
+    assert(err = 0);
+}
+
```

Comment:
in manual to avcodec_close in avcodec.h there is a warning, that "Opening and closing a codec context multiple times is not supported"

Reply:
Indeed, you're right... Anyway I found a much better way to clear the context for both ffmpeg and libatrac9.